code/game/turfs/closed/walls.dm

```diff
@@ -32,6 +32,12 @@
 
 	var/list/dent_decals
 
+/turf/closed/wall/Initialize(mapload)
+	..()
+	if(is_station_level(z))
+		GLOB.station_turfs += src
```

Comment:
They should remove themselves from this list on `Destroy()` instead of leaving an invalid reference.

Reply:
done